pennylane/transforms/__init__.py

```diff
@@ -192,17 +161,60 @@
     ~transforms.poly_extrapolate
     ~transforms.richardson_extrapolate
 
-Transforms core
----------------
+Other transforms
+~~~~~~~~~~~~~~~~
+
+These transforms use the :func:`pennylane.transforms.core.transform` decorator and can be used on
```

Comment:
@trbromley @rmoyard are we planning to have this function be available as `pennylane.transform`?

Reply:
I would be happy to have it as `pennylane.transform`, for the moment it is `pennylane.transforms.transform`.

Reply:
Yes please, which is also what we do with the old transforms.

Reply:
Top level added :+1: